Makefile

```diff
@@ -1,8 +1,26 @@
-build:
+.PHONY: help build init upgrade/theme
+
+default: help
+
+help: # Show help for each of the Makefile recipes.
+	@echo ""
+	@echo "██╗  ██╗ █████╗ ██████╗ ██████╗ ██╗    ██╗ █████╗ ██████╗ ███████╗     ██████╗██╗████████╗██╗   ██╗"
+	@echo "██║  ██║██╔══██╗██╔══██╗██╔══██╗██║    ██║██╔══██╗██╔══██╗██╔════╝    ██╔════╝██║╚══██╔══╝╚██╗ ██╔╝"
+	@echo "███████║███████║██████╔╝██║  ██║██║ █╗ ██║███████║██████╔╝█████╗      ██║     ██║   ██║    ╚████╔╝ "
+	@echo "██╔══██║██╔══██║██╔══██╗██║  ██║██║███╗██║██╔══██║██╔══██╗██╔══╝      ██║     ██║   ██║     ╚██╔╝  "
+	@echo "██║  ██║██║  ██║██║  ██║██████╔╝╚███╔███╔╝██║  ██║██║  ██║███████╗    ╚██████╗██║   ██║      ██║   "
+	@echo "╚═╝  ╚═╝╚═╝  ╚═╝╚═╝  ╚═╝╚═════╝  ╚══╝╚══╝ ╚═╝  ╚═╝╚═╝  ╚═╝╚══════╝     ╚═════╝╚═╝   ╚═╝      ╚═╝   "
+	@echo ""
+	@echo "                                                                           contact@hardwarecity.org"
+	@echo "Usage: make [target]"
+	@grep -E '^[a-zA-Z0-9 -]+:.*#'  Makefile | sort | while read -r l; do printf "* \033[1;32m$$(echo $$l | cut -f 1 -d':')\033[00m:$$(echo $$l | cut -f 2- -d'#')\n"; done
+	@echo ""
+
+build: # Build the site.
 	hugo --minify --environment production
 
-init:
-	git submodule update --init 
+init: # Initialize the theme submodule.
+	git submodule update --init
 
-upgrade/theme:
-	git submodule update --remote --merge
\ No newline at end of file
+upgrade/theme: # Upgrade the theme submodule.
```

Comment:
@dvcorreia , este `upgrade/theme` faz parte do projecto do HUGO, ou foi adicionar por ti?

o `make help` não consegue detectar a descrição deste script por causa da `/`, acho eu.

Se o `upgrade/theme` faz parte do HUGO, então eu mudo o regex, mas se foi adicionado por ti, o que achas de mudarmos para `upgrade-theme`?

<img width="823" alt="Screenshot 2023-11-08 at 23 28 20" src="https://github.com/HardwareCity/hardwarecity.github.io/assets/2766648/48aa3533-82ca-46ba-b3a7-0cb6fbf09465">

Reply:
fui eu. força, muda! se não funciona é mudar.